neutrons/RefRed
# added utility function to temporarily amend config for reduction

**Description of work.**

Added utility function to temporarily amend Mantid's user properties to ensure SNS Search Data Archive is set. This utility function was used in a few places to ensure RefRed was able to properly load a configuration and run a reduction. 

**To Test:**

- the tester should start Mantid's workbench and then `File --> Manage User Directories` and set the `Seach Data Archive` to `Off` and close the `Manage User Directories` windows.
- start RefRed and reduce any configuration
- reopen Mantid's workbench and then `File --> Manage User Directories`. Verify the `Seach Data Archive` is turned off.
